### PR TITLE
feat: Add a "stats for nerds" page

### PR DESCRIPTION
**Pull request by tofumatt**
Adds a bit of "how it's built" info to the site, largely to be used from a blog post that'll link to this.

I think this is a generally good/fun habit for us to get into for our products, especially ones that aren't entirely open-source. Let me know if you want to add anything to this—especially from the "design tools"/design approach side-of-things!

### CODE REVIEW

File: src/content/pages/for-nerds.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Credits
```
Comment (sarahmonster) [inline src/content/pages/for-nerds.mdx:2]:
```suggestion
title: Under the hood
```

Comment (sarahmonster) [reply src/content/pages/for-nerds.mdx:2]: Or just "How it's built"?

Also, I feel like 'for-nerds' is a weird URL. Instead, I'd go with `under-the-hood` or `how-its-made` or `technical-details` or even just `for-the-curious`.

File: src/content/pages/for-nerds.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+import Panel from 'components/Panel';
+
+<Panel>
+
+## How is Mic Drop built?
+
+Mic Drop is a macOS application written _entirely_ in Swift and SwiftUI. It uses **Core Audio** to manage mute/volume state. Mic Drop requires Input Monitoring permissions to listen for global keyboard shortcuts—particularly double-tap, modifier-only shortcuts. ⌨️
```
Comment (sarahmonster) [inline src/content/pages/for-nerds.mdx:11]:
```suggestion
Mic Drop is a macOS application written _entirely_ in Swift and SwiftUI. It uses Core Audio to manage mute and volume states. Mic Drop requires Input Monitoring permissions to listen for global keyboard shortcuts—particularly double-tap, modifier-only shortcuts. ⌨️
```

....why "particularly"?

File: src/content/pages/for-nerds.mdx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Mic Drop _doesn't_ use audio recording/microphone access APIs, so it can **never hear you or listen to you**. The way Mic Drop is implemented, it only has access to your audio device IDs, their mute status, and their volume level. It never has access to the audio stream being received by your microphone, even if it's on. 👌🏻
```
Comment (sarahmonster) [inline src/content/pages/for-nerds.mdx:13]:
```suggestion
We purposefully didn't make use of any audio recording or microphone access APIs. This way, we can be certain that Mic Drop can **never hear you or listen to you**. The way it's implemented, it only has access to your audio device IDs, their mute status, and their volume level. It never has access to the audio stream being received by your microphone, even if it's on. 👌🏻
```

File: src/content/pages/for-nerds.mdx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+**Mic Drop requires macOS 10.15 Catalina**, because we wanted to try SwiftUI. It's kinda cool and feels like "React for Swift", but with many levels of air quotes. SwiftUI has promise, but it's very early-stages right now. 🤷🏻‍♂️
+
+## Open source libraries
+
+For open-source libraries in Mic Drop, see the [credits page](/credits).
```
Comment (sarahmonster) [inline src/content/pages/for-nerds.mdx:19]:
```suggestion
For open-source libraries used in Mic Drop, see the [credits page](/credits).
```

File: src/content/pages/for-nerds.mdx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+We use a customised version of both [KeyHolder](https://github.com/octopusthink/KeyHolder) and [Magnet](https://github.com/octopusthink/Magnet), both of which are available on GitHub.
```
Comment (sarahmonster) [inline src/content/pages/for-nerds.mdx:21]:
```suggestion
We use a customised version of both [KeyHolder](https://github.com/octopusthink/KeyHolder) and [Magnet](https://github.com/octopusthink/Magnet), both of which are available on GitHub. (Feel free to use them!)
```

File: src/content/pages/for-nerds.mdx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+## How is this website built?
+
+We built this website with Gatsby and our own [Ghost Ship](https://github.com/octopusthink/ghost-ship/). The [source code for this site is available on GitHub](https://github.com/octopusthink/getmicdrop.com/).
```
Comment (sarahmonster) [inline src/content/pages/for-nerds.mdx:25]:
```suggestion
We built this website with Gatsby and our Gatsby starter theme ([Ghost Ship](https://github.com/octopusthink/ghost-ship/)) and design system ([Nautilus](https://github.com/octopusthink/nautilus)). The [source code for this site is available on GitHub](https://github.com/octopusthink/getmicdrop.com/).
```

File: src/content/pages/for-nerds.mdx [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+## Open source libraries
+
+For open-source libraries used in Mic Drop, see the [credits page](/credits).
+
+We use a customised version of both [KeyHolder](https://github.com/octopusthink/KeyHolder) and [Magnet](https://github.com/octopusthink/Magnet), both of which are available on GitHub. (Feel free to use them!)
+
+## How is this website built?
+
+We built this website with Gatsby and our own [Ghost Ship](https://github.com/octopusthink/ghost-ship/). The [source code for this site is available on GitHub](https://github.com/octopusthink/getmicdrop.com/).
```
Comment (sarahmonster) [inline src/content/pages/for-nerds.mdx:25]:
```suggestion
We built this website with Gatsby, our own Gatsby starter theme ([Ghost Ship](https://github.com/octopusthink/ghost-ship/)) and our design system ([Nautilus](https://github.com/octopusthink/nautilus)). The [source code for this site is available on GitHub](https://github.com/octopusthink/getmicdrop.com/).The [source code for this site](https://github.com/octopusthink/getmicdrop.com/) is available on GitHub if you're curious and/or want to see how many times we changed the shade of red used. (Spoiler alert: however many times you think sounds reasonable, it's probably more.)
```

File: src/content/pages/for-nerds.mdx (hunk 2 above)
Comment (tofumatt) [reply src/content/pages/for-nerds.mdx]: Because not using those shortcuts you can get away with not asking for permissions 😆

**Pull request merged**